DSE_6.8_Release_Notes.md

```diff
@@ -1,6 +1,31 @@
 # Release notes for DataStax Enterprise 6.8
 DSE 6.8.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version. 
 
+# Release notes for 6.8.16
+
+7 October 2021
+
+## 6.8.16 DSE Cassandra
+
+* The Change Data Capture (CDC) is now near real time as in Cassandra 4.x. Active commitlog segments can be processed from the CDC raw directory defined in your {{cassandra.yaml}} without flushing tables. It also allows you to read the last mutation form the most recent commitlog file. See the Datastax Change Data Capture documentation page for more details. (DSP-21992)
```

Comment:
Will {{cassandra.yaml}} render properly? I _think_ it'd be better to do it as `cassandra.yaml` (surrounding with single quotes)

Reply:
Tiago, thanks. If not a hassle, can you insert the documentation URL on L10?  

[documentation](https://docs.datastax.com/en/dse/6.8/dse-admin/datastax_enterprise/config/configCDCLogging.html)

The updated topic will be in place shortly after 6.8.16 GAs.